src/pocketmine/command/FormattedCommandAlias.php

```diff
@@ -59,7 +59,7 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
 		}
 
 		foreach($commands as $command){
-			$result |= Server::getInstance()->dispatchCommand($sender, $command, true);
+			$result &= Server::getInstance()->dispatchCommand($sender, $command, true);
```

Comment:
It now only returns true if all commands return true. I think this makes more sense than returning true if at least one command returns true. The return value is unused anyways, but if it were, it makes sense that the entire command alias would be a failure if at least one of the commands failed.

Reply:
yes, it is unused... so why change it?

Reply:
My reasoning was this: it used each command's return value in the first place (it didn't always return true, only if at least one command did). Since that return value is unused, you're right, it *is* useless. I figured, however, that if we're gonna have a return value that's decided by an algorithm, might as well have that algorithm make sense. I guess it'd also make sense to have the command return true every time.

Reply:
This change is unnecessary. Please remove it.